### DISCUSSION TITLE
Uncaught BadMethodCallException

### DISCUSSION THREAD

**Discussion by calebsg0** [YOURLS/YOURLS#3323] [General]
Since upgrading to 1.9 I am receiving a PHP fatal error:
[04-May-2022 13:29:02 UTC] PHP Fatal error: Uncaught BadMethodCallException: Class 'YOURLS\Database\YDB' does not have a method 'get_var' in /home/oyfltsiv/public_html/includes/vendor/aura/sql/src/AbstractExtendedPdo.php:113
Stack trace:
```
#0 /home/oyfltsiv/public_html/user/plugins/case-insensitive/plugin.php(26): Aura\Sql\AbstractExtendedPdo-&gt;__call('get_var', Array)
#1 /home/oyfltsiv/public_html/includes/functions-plugins.php(229): insensitive_keyword_is_taken(false, 'caleb')
#2 /home/oyfltsiv/public_html/includes/functions-shorturls.php(435): yourls_apply_filter('shunt_keyword_i...', false, 'caleb')
#3 /home/oyfltsiv/public_html/yourls-loader.php(46): yourls_keyword_is_taken('caleb')
#4 {main}
thrown in /home/oyfltsiv/public_html/includes/vendor/aura/sql/src/AbstractExtendedPdo.php on line 113
```

My install has been working perfectly for several years! This error arises when I enter an existing shortlink into my web browser and the browser returns a blank page. E.g., http:/oyf.link/caleb should redirect to https://www.onlyyouforever.com/our-team/caleb-simonyi-gindele/

The admin interface works fine!

**Comment by calebsg0** [comment]
UPDATE: I looked more closely at the case-insensitive plugin and solved this error by removing the plugin:
https://github.com/adigitalife/yourls-case-insensitive

And replacing it with:
https://github.com/IMExperts/yourls-case-insensitive

I did so after seeing the issue posted here:
https://github.com/adigitalife/yourls-case-insensitive/issues/8

**Reply by ozh** [reply]
Thanks for reporting and for sending a notice to the plugin dev ;)
